.gitignore

```
@@ -160,3 +160,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+.bash_history
+.jupyter/lab/workspaces/default-37a8.jupyterlab-workspace
```

```suggestion
.jupyter/
```
This should ignore everything inside the .jupyter folder.